### DISCUSSION TITLE
NativeShare Error

### DISCUSSION THREAD

**Discussion by jaaronlee** [yasirkula/UnityNativeShare#120] [Q&A] [answered]
Hello, I am getting an error (does not exist in current context) with the code below. Am I missing a using statement, or are the utility functions no longer operational? Let me know if you need any further information.

This line is working elsewhere in the code, so I think the plugin is properly installed:
UnityNativeSharingHelper.ShareScreenshotAndText(text, screenshotPath, false, "Select App To Share With");

using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using System;
using System.IO;
using UnityEngine.UI;
using UnityEngine.Events;
using UnityNative;
using UnityNative.Sharing.Example;
using UnityNative.Sharing;

public class ShareManager : MonoBehaviour
{
public GameObject facebookButton;

void start(){

        if (!NativeShare.TargetExists("com.facebook.android"))
        {
            facebookButton.SetActive(false);
        }
}
}

**Comment by yasirkula** [comment, marked as answer]
This plugin doesn't have UnityNativeSharing class. You've probably imported [this repository](https://github.com/NicholasSheehan/Unity-Native-Sharing) whereas you should've imported NativeShare via [one of these methods](https://github.com/yasirkula/UnityNativeShare#installation).

**Reply by jaaronlee** [reply]
Works like a charm! Thank you for the quick help!
